stories/Welcome.mdx

```diff
@@ -1,7 +1,138 @@
-import './styles.css';
+import './welcome-styles.css';
+import gridironLogo from './assets/gridironSurvivorLogo.svg';
+import appwriteLogo from './assets/appwriteLogo.svg';
+import blogRecorderLogo from './assets/blogRecorderLogo.svg';
+import frontendMentorLogo from './assets/frontendMentorLogo.svg';
+import gitkrakenLogo from './assets/gitkrakenLogo.svg';
+import pastelLogo from './assets/pastelLogo.svg';
+import Image from 'next/image';
 
-<h1>Welcome</h1>
+<html>
+  <body class="dark bg-background text-muted-foreground max-w-screen-xl mx-auto text-pretty flex flex-col space-y-36 pb-36">
+    <header class="header flex justify-center align-middle">
+      <a href="https://www.gridironsurvivor.com" target="_blank" rel="noreferrer" class="w-2/6 h-fit">
+        <Image class="w-full" src={gridironLogo} alt='Gridiron Survivor Football Logo' />
+      </a>
+    </header>
+    <section class="hero space-y-12">
+      <h1 class="pl-16">What is Gridiron Survivor?</h1>
+      <p class="text-xl text-pretty max-w-screen-lg mx-auto">
+        Gridiron Survivor is the ultimate pick'em league app. Track your picks, see how you stack up against your friends, and hope to be crowned champion! Developed by a talented team of junior developers using technologies such as Next.js, React, and Tailwind CSS, Gridiron Survivor is the perfect tool for any football fan looking to take their pick'em game to the next level.
+        <br />
+        Gridiron Survivor is currently in its first Alpha for the 2024-2025 NFL Season! 1st, 2nd, and 3rd place winners of the season get spoiled with prizes from our generous sponsors.
```

Comment:
nit: I'm not sure how I feel about the choice of using spoiled here. Also unless we find a way to automate things like 2024-2025 and first Alpha, this will need to be maintained. If it's possible to do in Storybook, we could make a partial or something similar to that so we could automate this just a bit.

Reply:
Good points here. The information in the second `<p>` is probably not needed, either. I wouldn't mind if we just omit it.

Reply:
Keep it for now tentatively, but we could remove it entirely later after we are done promoting this challenge.

Reply:
Removed the second paragraph after altering the copy on the first one.